.circleci/config.yml

```diff
@@ -4,19 +4,17 @@ orbs:
   ft-snyk-orb: financial-times/ft-snyk-orb@0
 
 references:
-
-  container_config_carbon: &container_config_carbon
+  container_config_fermium: &container_config_fermium
     working_directory: ~/project/lambda-logger
     docker:
-      - image: circleci/node:8.10.0
+      - image: circleci/node:fermium # codename for nodejs active LTS v14
```

Comment:
I'd probably avoid using the codename as they're not widely known - if the comment is necessary then it's too complex 😄 

`node:14` should do the trick 👍